apps/ui/src/components/utils/spinner.tsx

```diff
@@ -8,6 +8,7 @@ export function Spinner({
   return (
     <svg
       aria-hidden="true"
+      data-testid="spinner"
```

Comment:
Why does it have to be `data-testid` ? Could we use the standard `id` instead? 

We have a file called 'testing.ts' that exports a bunch of contstant's. 

https://github.com/rocket-connect/graphql-debugger/blob/main/apps/ui/src/testing.ts